packages/aws-client/src/SQS.ts

```diff
@@ -160,7 +160,7 @@ export const receiveSQSMessages = async (
 export const parseSQSMessageBody = (
   message: SQSRecord | AWS.SQS.Message
 ): { [key: string]: any } =>
-  JSON.parse(get(message, 'Body', get(message, 'body', '{}')));
+  JSON.parse(get(message, 'Body', get(message, 'body')) || '{}');
```

Comment:
Nit: we should use nullish coalescing here instead of || generally as a pattern. 

Something like JSON.parse(message?.Body?.body ?? '{}')

Reply:
Apologies, I misread that on the first pass.   

Something like `get(message, 'Body', get(message, 'body')) ?? '{}'`?    The important bit is `||` vs `??`.